src/torznab.ts

```diff
@@ -115,8 +115,12 @@ function createTorznabSearchQuery(name: string, caps: Caps) {
 		return {
 			t: "tvsearch",
 			q: cleanseSeparators(match.groups.title),
-			season: extractNumber(match.groups.season),
-			ep: extractNumber(match.groups.episode),
+			season: match.groups.season
+				? extractNumber(match.groups.season)
+				: match.groups.year,
+			ep: match.groups.episode
+				? extractNumber(match.groups.episode)
+				: `${match.groups.month}/${match.groups.day}`,
```

Comment:
is this how daily shows are supposed to be handled in torznab?

Reply:
This is how Prowlarr shows the searches being made to trackers.